config.go

```diff
@@ -99,6 +99,7 @@ type config struct {
 	RPCKey             string        `long:"rpckey" description:"File containing the certificate key"`
 	RPCMaxClients      int           `long:"rpcmaxclients" description:"Max number of RPC clients for standard connections"`
 	RPCMaxWebsockets   int           `long:"rpcmaxwebsockets" description:"Max number of RPC websocket connections"`
+	RPCQuirks          bool          `long:"rpcquirks" description:"Mirror some JSON-RPC quirks of the reference implementation -- NOTE: Discouraged unless interoperability issues need to be worked around"`
```

Comment:
Would you mind making this say `of Bitcoin Core`?  It's more descriptive for people that are unaware.